tool/llpcAutoLayout.cpp

```diff
@@ -53,6 +53,12 @@ using namespace llvm;
 using namespace Llpc;
 using namespace SPIRV;
 
+struct ResNodeSet
```

Comment:
Add comment

// Represent the info of resource node set.

Reply:
Superseded by your comment below on the same line.